src/app/(post-login)/submission/components/ProgramMenu.tsx

```diff
@@ -172,28 +202,76 @@ const MenuContent = ({ programName }: { programName: string }) => {
 
 	return (
 		<>
-			{/** Register Samples */}
-			<Link href={`/submission/program/${programName}/registration`}>
-				<MenuItem
-					level={3}
-					content={
-						<StatusMenuItem>
-							Register Samples
-							{statusIcon}
-						</StatusMenuItem>
-					}
-					selected={pathnameLastSegment === 'registration'}
-				/>
+			{/** Dashboard */}
+			<Link
+				as={PROGRAM_DASHBOARD_PATH.replace(PROGRAM_SHORT_NAME_PATH, programName)}
+				href={PROGRAM_DASHBOARD_PATH}
+			>
```

Comment:
why is adding `as` prop needed?

Reply:
This is direct from Platform, I can look into it

Reply:
Gotcha. So the `as` is usually when you have middleware of some kind. You want the functionality to be to specific urls, but need it displayed as single. The example they give in their docs is something like `/auth/dashboard` vs `/dashboard` but you still want the link as `/dashboard`

Reply:
I will remove `as` for now and update `href` to `href={PROGRAM_MANAGE_PATH.replace(PROGRAM_SHORT_NAME_PATH, programName)}` for each path

Reply:
This is rendering correctly